Makefile

```diff
@@ -15,7 +15,7 @@
 #-------------------------------------------------------------------------------
 # General build options
 # Change this parameter while releasing
-MAIN_VERSION := 0.4.0
+MAIN_VERSION := 0.6.0
```

Comment:
```suggestion
MAIN_VERSION := $(shell git describe --tags `git rev-list --tags --max-count=1` | sed 's/^v//')
```

Reply:
> 
Is is correct when we release a new version? For example if we are going to release v0.7.0, there is no v0.7.0 tag.

Reply:
But we always 1) push new tag 2) draft new release, is there any unexpected workflow?

Reply:
> But we always 1) push new tag 2) draft new release, is there any unexpected workflow?

I usually do that at the same time
![image](https://github.com/user-attachments/assets/9591091a-cef8-4474-87ca-70a2d65ae095)

Reply:
> I usually do that at the same time 

You mean 1) create new tag on github 2) build from un-taged latest commit on local 3) upload artifacts?

Reply:
> > I usually do that at the same time
> 
> You mean 1) create new tag on github 2) build from un-taged latest commit on local 3) upload artifacts?

1. build from un-taged latest commit on local and upload artifacts 2. create new tag on github and release at the same time